prometheus/prometheus.yml

```diff
@@ -46,5 +48,9 @@ scrape_configs:
         target_label: __param_target
       - source_labels: [__param_target]
         target_label: instance
+      - source_labels: [mib]           # Allow per-target MIB override
+        target_label: __param_module   # via 'mib' label
+      - target_label: mib
+        replacement: ''                # Remove the `mib` label
```

Comment:
All this was so that I could run a few legacy routers with the old MIB (which I renamed to `ddwrt_legacy`) and a few with the new.  The legacy routers got an extra `mib: ddwrt_legacy` label, and when I switched them over to the new SNMP config, that label was removed.  The bit about removing the `mib` label is so that I could do this transparently w/out creating different timeseries (due to label differences).

In hindsight, maybe not necessary, but I thought it was a handy way to migrate from v1 of a MIB to v2 of a MIB a little at a time.